updater/internal/updater/updater.go

```diff
@@ -16,10 +16,12 @@
 package updater
 
 import (
+	"bytes"
```

Comment:
4. Normal installs of the 1.80 versions seem fine, with Group=bdot in the service file.